docs/resources/destination_adobe_analytics.md

```diff
@@ -0,0 +1,235 @@
+
+### Read-Only
+
+- `created_at` (String) Time when the resource was created, in ISO 8601 format.
+- `id` (String) The ID of this resource.
+- `updated_at` (String) Time when the resource was last updated, in ISO 8601 format.
+
+<a id="nestedblock--config"></a>
+### Nested Schema for `config`
+
+Required:
+
+- `report_suite_ids` (String) Enter your Report Suite ID(s). You can add multiple report suite ids by separated by commas.
+
+Optional:
+
+- `context_data_mapping` (List of Object) You can map Rudder Context data to Adobe Context Data (see [below for nested schema](#nestedatt--config--context_data_mapping))
+- `context_data_prefix` (String) Enter your prefix to add before all contextData property.
+- `custom_props_mapping` (List of Object) You can map Rudder Properties to Adobe Custom properties (see [below for nested schema](#nestedatt--config--custom_props_mapping))
+- `drop_visitor_id` (Boolean) Check to Drop Visitor Id.
+- `e_var_mapping` (List of Object) You can map Rudder Properties to Adobe eVars (see [below for nested schema](#nestedatt--config--e_var_mapping))
+- `event_filtering` (Block List, Max: 1) This option allows you filter the events you want to send to Adobe Analytics. (see [below for nested schema](#nestedblock--config--event_filtering))
+- `event_merch_event_to_adobe_event` (List of Object) You can map Rudder Events to Adobe Merchandise events. (see [below for nested schema](#nestedatt--config--event_merch_event_to_adobe_event))
+- `event_merch_properties` (List of String) Currency/Incremental properties to add to merchandise events at event level
+- `events_to_types` (List of Object) You can map your Rudder video events with types of Video Events (see [below for nested schema](#nestedatt--config--events_to_types))
+- `heartbeat_tracking_server_url` (String) Enter your Heartbeat Tracking Server URL
+- `hier_mapping` (List of Object) You can map Rudder Properties to Adobe Hierarchy properties (see [below for nested schema](#nestedatt--config--hier_mapping))
+- `list_delimiter` (List of Object) You can map your Rudder Property with Delimiters for list properties (see [below for nested schema](#nestedatt--config--list_delimiter))
+- `list_mapping` (List of Object) You can map Rudder Properties to Adobe list properties (see [below for nested schema](#nestedatt--config--list_mapping))
+- `marketing_cloud_org_id` (String) Enter your Marketing Cloud Organization Id.
+- `mobile_event_mapping` (List of Object) You can map Rudder Mobile events. (see [below for nested schema](#nestedatt--config--mobile_event_mapping))
+- `no_fallback_visitor_id` (Boolean) Check to enable no Fallbacks for Visitor ID
+- `onetrust_cookie_categories` (Block List, Max: 1) Specify the OneTrust category name for mapping the OneTrust consent settings to RudderStack's consent purposes. (see [below for nested schema](#nestedblock--config--onetrust_cookie_categories))
+- `page_name_fallback_tostring` (Boolean) Check to allow Page Name Fallback to Screen
```

Comment:
Confirming if it should be screen or string?

Reply:
It should be screen, I referenced it from the UI.